my-app/src/utils/constants/data.tsx

```diff
@@ -0,0 +1,8 @@
+
+import ImagePath from ".";
+export const iconsData = [
+  { src: ImagePath.image, alt: 'Image not found' },
+  { src: ImagePath.image1, alt: 'Image not found' },
+  { src: ImagePath.image2, alt: 'Image not found' },
+  { src: ImagePath.image3, alt: 'Image not found' }
+];
```

Comment:
place them in the constants file no need of separate data file

Reply:
changed